### PR TITLE
Minimal Occupancy/Capacity added

### PR DESCRIPTION
**Pull request by ue71603**
based on SIRI. When new Siri is loaded, then some changes are obsolete. Only minimal implementation (otherwise half of the new Siri would be needed here).

Be ware that mainly the change in OJP_JourneySupport is important. When we do the update from Siri, then a lot of things will be better an the full SIRI version is available.
As we don't distinguish DepartureStatusGroup I added it here. 

Fixes: https://github.com/VDVde/OJP/issues/250

### CODE REVIEW

File: siri/siri_model/siri_journey-v2.0.xsd [hunk 1]
```diff
@@ -897,4 +897,192 @@ Rail transport, Roads and road transport
...
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="BicycleRackCapacity" type="xsd:nonNegativeInteger" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>The number of bicycle racks on the vehicle.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+		</xsd:sequence>
+	</xsd:group>
+	<!--will be updated with new SIRI	<xsd:group name="OccupancyScopeFilterGroup">
```
Comment (skinkie) [inline siri/siri_model/siri_journey-v2.0.xsd:948]: Is the stuff below what we want?

**Comment by ue71603** [comment]
blocked by upstream-siri

**Comment by ue71603** [comment]
First class is missing.

File: OJP/OJP_JourneySupport.xsd [hunk 2]
```diff
@@ -353,6 +353,16 @@
 					<xs:documentation>Alighting will not be allowed at this stop of this journey.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:choice>
+				<xs:sequence>
+					<xs:element ref="siri:ExpectedDepartureOccupancy" minOccurs="0" maxOccurs="unbounded"/>
+					<xs:element ref="siri:ExpectedDepartureCapacities" minOccurs="0" maxOccurs="unbounded"/>
+				</xs:sequence>
+				<xs:sequence>
+					<xs:element ref="siri:RecordedDepartureOccupancy" minOccurs="0" maxOccurs="unbounded"/>
```
Comment (Aurige) [inline OJP/OJP_JourneySupport.xsd:362]: What does "recorded" means for a journey planner ?... the answer is always expected to be about journeys is in some future, isn't it ?

**Comment by ue71603** [comment]
Problem with 1 and second class to be solved.

**Comment by ue71603** [comment]
@skinkie  We need the new SIRI before that. Occupancy in SIRI 2.1 does support first and second class.

**Comment by ue71603** [comment]
Ok. Finalise

File: siri/siri_model/siri_journey-v2.0.xsd (hunk 1 above)
Comment (ue71603) [reply siri/siri_model/siri_journey-v2.0.xsd]: jupp

File: OJP/OJP_JourneySupport.xsd [hunk 3]
```diff
@@ -353,6 +353,16 @@
 					<xs:documentation>Alighting will not be allowed at this stop of this journey.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:choice>
+				<xs:sequence>
+					<xs:element ref="siri:ExpectedDepartureOccupancy" minOccurs="0" maxOccurs="unbounded"/>
```
[the remark this answers is not in the document]
Comment (ue71603) [reply OJP/OJP_JourneySupport.xsd]: example done

**Comment by ue71603** [comment]
description in spec in 5.3.4, 5.4.6, 7 (some sections and very briefly), 8.8.1

**Review by skinkie — APPROVED** [review]

**Review by Aurige — APPROVED** [review]

**Pull request merged**